sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1045,9 +1045,10 @@ object SQLConf {
     buildConf("spark.sql.execution.arrow.enabled")
       .internal()
```

Comment:
This conf will be mentioned in the doc `https://github.com/apache/spark/pull/19575`.  I think this is not qualified as an internal conf.

Reply:
Could you change it in this PR and use this JIRA number https://issues.apache.org/jira/browse/SPARK-23238

Reply:
Yup.